filefinder/_filefinder.py

```diff
@@ -384,7 +394,7 @@ def find_paths(
         )
 
     def find_files(
-        self, keys=None, on_parse_error="raise", _allow_empty=False, **keys_kwargs
+        self, keys=None, *, on_parse_error="raise", _allow_empty=False, **keys_kwargs
```

Comment:
This change is unrelated but I thought not worth a separate PR (strictly it is a breaking change)

Reply:
could put it into the CHANGELOG but 🤷‍♀️